frb_codegen/src/lib.rs

```diff
@@ -34,7 +34,7 @@ use error::*;
 
 pub fn frb_codegen(config: &config::Opts, all_symbols: &[String]) -> anyhow::Result<()> {
     let dart_root = config.dart_root_or_default();
-    ensure_tools_available(&dart_root)?;
+    ensure_tools_available(&dart_root, config.skip_deps_check)?;
```

Comment:
uber-nit: Some people think bool flags should be positive, i.e. `enable_deps_check`. So maybe `ensure_tools_available(!config.skip_deps_check)` and `fn ensure_tools_available(enable_deps_check: bool)`. But anyway current approach is acceptable to me

Reply:
I don't mind changing it, but I've actually followed the convention used for another arg: [skip_add_mod_to_lib](https://github.com/fzyzcjy/flutter_rust_bridge/blob/dc5d3d994c07c60b702d491be5581b3785a5c986/frb_codegen/src/config.rs#L77).
So please confirm me again if you would like me to do the change anyway @fzyzcjy .

Reply:
I see. So no need to change :)